packages/plugin-core/.vscode/launch.json

```diff
@@ -143,6 +143,26 @@
       ],
       "outFiles": ["${workspaceFolder}/out/src/test/**/*.js"]
     },
+    {
```

Comment:
if you look at the task `      "name": "Extension Integ Tests - File Remote",`  - this is actually running the integ test with extension. feel free to rename it instead of creating a new one